src/playbacks/hls/hls.js

```diff
@@ -119,6 +133,22 @@ export default class HLS extends HTML5VideoPlayback {
     this.trigger(Events.PLAYBACK_SETTINGSUPDATE)
   }
 
+  onError(evt, data) {
+    if (data && data.fatal) {
+      switch (data.type) {
+        case Hls.ErrorTypes.NETWORK_ERROR:
+          hls.startLoad()
```

Comment:
@tjenkinson: I'm unsure how to add a delay because things can happen between starting the delay and recovering which might cause the recover call to be invalid/bad.